tests/setup_conductor.js

```diff
@@ -35,7 +34,7 @@ async function start_conductor() {
 
 async function stop_conductor(timeout) {
   console.log("Closing Conductor...");
-  exec("make stop-conductor", (error, stdout, stderr) => {
+  await exec("make stop-conductor", (error, stdout, stderr) => {
```

Comment:
I think what you want here is `execSync` and importing it from `child_process`. `exec` doesn't actually return a promise so this is a useless `await`